enhancements/ingress/optional-ingress-hypershift.md

```diff
@@ -0,0 +1,323 @@
+but the component owner (operator) won't be able to access the secret or rely on the route implementation.     
+This may be misleading for the cluster instance administrator. However the component routes were never fully implemented on HyperShift.
+For instance, the authentication operator which is supposed to implement the authentication component route is not used at all. Therefore no entry is added into the `.status.componentRoutes` of the Ingress Config, which, according to the API contract, prevents the cluster admin from customizing the component route.
+
+#### Ingress to route controller
+
+[The Kubernetes Ingress API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#ingress-v1-networking-k8s-io) on OpenShift is implemented through translation into routes.
+[The ingress-to-route controller](https://github.com/openshift/route-controller-manager#ingress-to-route-controller) is the component responsible for this functionality. When the default ingress controller is not present, the routes are left without serving, resulting in the ingresses also being unserved. On HyperShift this can be mitigated by [any ingress controller implementation](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/).
+In the context of this enhancement, [AWS Load Balancer Operator](/enhancements/ingress/aws-load-balancer-operator.md) is the expected implementation.
+
+### Drawbacks
+
+In the absence of the default ingress controller, the Route API is left unserved. OpenShift components or workloads can create routes, but there is no available controller to support them.
+An important consequence of this is the impact on the ingress API which relies on [the ingress to route translation](#ingress-to-route-controller). All this may be misleading for the users.   
+One mitigation to this drawback is that the route status will reflect that no ingress controller has admitted the route. This should be considered an indicator of the potential risks associated with using the route.
+
+## Design Details
+
+### Open Questions [optional]
+
+N/A
+
+### Test Plan
+TBD
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+N/A
+
+#### Tech Preview -> GA
+N/A
+
+#### Removing a deprecated feature
+N/A
+
+### Upgrade / Downgrade Strategy
+
+Same as [the component selection strategy](/enhancements/installer/component-selection.md#upgrade--downgrade-strategy).     
+Note that:
+- the capabilities [cannot disabled post install](/enhancements/installer/component-selection.md#capabilities-cannot-be-uninstalled)
+- the capabilities can be [implicitly enabled](/enhancements/installer/component-selection.md#updates) (e.g. cluster version with `None` baseline upgrades to the version which moves some of the core payloads to a capability)
+
+### Version Skew Strategy
+N/A
+
+### Operational Aspects of API Extensions
+N/A - no api extensions are being introduced in this proposal
```

Comment:
This would be good place to explain how to check whether the ingress capability is enabled on a cluster.  Maybe also re-iterate that the ingress operator runs on the hosted control-plane, so checking for its presence on the guest cluster isn't sufficient.  However, the cluster-admin could check for the "ingress" clusteroperator or the various resources related to metrics and RBAC to determine whether the ingress operator is running, right?  The expected output of `oc -n openshift-ingress-operator get all` could be helpful.

Reply:
I created a HyperShift hosted cluster using the clusterbot. It turned out that `ClusterVersion` is available for the user (at least for the cluster admin). I added "Failure modes" and "Support procedures" chapters to show how to check the ingress capability.